raiden_contracts/constants.py

```diff
@@ -29,9 +29,9 @@
 CONTRACT_DEPOSIT = "Deposit"
 
 # Timeouts
-TEST_SETTLE_TIMEOUT = 8
+TEST_SETTLE_TIMEOUT = 8 * 15  # Timeout in seconds
```

Comment:
Nitpick: this is somewhat awkward. What does `8 * 15` express? Are `15` seconds something special and then take it 8 times? I would have expected something like `2 * 60` or so to express 2 minutes or so. Else just the pure number. The same goes for the timeout value below.

Reply:
We were using 8 blocks as a timeout. However, in arbitrum, we can't be sure the time between each block is the same. So we add here the expected time between blocks, which is 15 seconds * 8 blocks